org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/commands/ClasspathItem.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Microsoft Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Microsoft Corporation - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.jdt.ls.core.internal.commands;
+
+import org.eclipse.core.runtime.IPath;
+
+public class ClasspathItem {
+
+	public final static String PROJECT = "Project";
+
+	public final static String CONTAINER = "Container";
+
+	public final static String JAR = "Jar";

```

Comment:
Not only jars can be package fragment roots, but also other projects or folders on the file system outside of a project.